pax-compiler/new-project-template/bin/run.rs

```diff
@@ -0,0 +1,24 @@
+use std::process::Command;
+use std::env;
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+
+    let pax_args = if args.is_empty() {
+        vec!["run", "--target=web"]
```

Comment:
Note that the CLI separately handles defaults for target, i.e. when a target isn't specified.  This introduces a "second source of truth" for default target, which is a latent headache for our future selves.  Not worth stopping merge.

Reply:
Ah I didn't realize. Will clean that up and push another small PR.